lucupy/observatory/gemini/geminiproperties.py

```diff
@@ -62,10 +76,27 @@ def determine_standard_time(resources: FrozenSet[Resource],
 
     @staticmethod
     def is_instrument(resource: Resource) -> bool:
+        """Checks if the resource is a Gemini instrument.
+
+        Args:
+            resource (Resource): A resource to be checked 
+
+        Returns:
+            bool: True if resource belong to a Gemini instrument.
+        """
         return resource in GeminiProperties.Instruments
 
     @staticmethod
     def acquisition_time(resource: Resource, observation_mode: ObservationMode) -> Optional[timedelta]:
+        """_summary_
+
+        Args:
+            resource (Resource): _description_
+            observation_mode (ObservationMode): _description_
+
+        Returns:
+            Optional[timedelta]: _description_
```

Comment:
Again, `_description_`? Not sure if this is a placeholder or has special meaning in the doc processor.